compiler/rustc_target/src/asm/mod.rs

```diff
@@ -885,6 +886,10 @@ impl InlineAsmClobberAbi {
                 "C" | "system" | "efiapi" => Ok(InlineAsmClobberAbi::LoongArch),
                 _ => Err(&["C", "system", "efiapi"]),
             },
+            InlineAsmArch::PowerPC => match name {
```

Comment:
You need to add `PowerPC64` here as well if both use the same calling convention. If the registers are different then you need a separate 32-bit and 64-bit `InlineAsmClobberAbi`.